src/excmds.ts

```diff
@@ -2511,19 +2531,26 @@ async function getnexttabs(tabid: number, n?: number) {
 
 // {{{ CMDLINE
 
-//#background_helper
-import * as controller from "@src/background/controller_background"
-
 /** Repeats a `cmd` `n` times.
-    Falls back to the last executed command if `cmd` doesn't exist.
+    If `cmd` doesn't exist, re-executes the last command.
     Executes the command once if `n` isn't defined either.
+
+    Note that "the last command" is a strange definition that depends
+    on some internal details of tridactyl. In general, `:repeat`
+    executed in the context of a tab will re-execute the last command
+    that was executed in that tab. However, many commands that touch
+    global state (for example, `:tabprev`) are also partially executed
+    in a global context, and if you manage to execute `:repeat` in
+    that global context it will repeat the last such global command.
```

Comment:
Could you add examples of when you might execute it in a global context? Is that the new tab page etc?

Reply:
...Huh. Apparently that's harder to do than I expected. Updated doc.